cmd/pipeline/main.go

```diff
@@ -543,12 +544,16 @@ func main() {
 
 			// Cluster Feature API
 			{
+				vaultClient, err := commonadapter.NewVaultClient()
```

Comment:
I think we should use `NewClientFromConfig` instead.

Reply:
https://github.com/banzaicloud/bank-vaults/blob/master/pkg/sdk/vault/client.go#L109